ControlRoomApplication/ControlRoomApplication/GUI/InputDialogs/CustomOrientationInputDialog.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using ControlRoomApplication.Constants;
+using ControlRoomApplication.Controllers;
+using ControlRoomApplication.Entities;
+
+namespace ControlRoomApplication.GUI
+{
+    public partial class CustomOrientationInputDialog : InputDialog
+    {
+        private double elevationHighLimit;
+        private double elevationLowLimit;
+        private double elevationPos;
+        private double azimuthPos;
+        private string[] values;
+        private Regex rx;
+
+        public CustomOrientationInputDialog(ref bool EnableSoftwareStops, ref string teleType, ref double maxElevationDegrees, ref double minElevationDegrees)
+        {
+            InitializeComponent();
+
+            rx = new Regex(@"[+-]?([0-9]+\.?[0-9]*|\.[0-9]+),[+-]?([0-9]+\.?[0-9]*|\.[0-9]+)");
```

Comment:
I just realized that this regex is incomplete because it allows input before or after. To fix this, add a `^` in front and a `$` after like this `^[+-]?([0-9]+\.?[0-9]*|\.[0-9]+),[+-]?([0-9]+\.?[0-9]*|\.[0-9]+)$` to restrict the input.

- [ ] Fix regex